### DISCUSSION TITLE
relative monthly triggers with interval more than one month

### DISCUSSION THREAD

**Discussion by TimurGinesin** [quartznet/quartznet#418] [General] [answered]
Seems Quartz.NET doesn’t support relative monthly triggers with interval more than one month. This case can be described by the following example:
“Run on first and last Monday of the month EVERY X MONTHS”. Such trigger cannot be implemented in Quartz.NET.  The only supported trigger is “Run on first and last Monday EVERY MONTH”.
Please, advice of how to configure requested "... EVEREY X MONTHS..." trigger. Thanks

**Comment by lahma** [comment, marked as answer]
I think this could achieved by using calendars in addition. In effect excluding months not wanted.
